src/config/logger.ts

```diff
@@ -43,45 +27,32 @@ const errorSerializer = (err) => err && {
 	request: requestSerializer(err.request)
 };
 
-const hashSerializer = (data: any): string | undefined => {
```

Comment:
We serialize at the stream level now since we need access to async feature flags.